### DISCUSSION TITLE
Would like to fix responsivness in dashboard

### DISCUSSION THREAD

**Discussion by felixdorn** [strapi/strapi#5094] [Legacy Topics]
- [x] **I have created my request on the Product Board before I submitted this issue**
- [x] **I have looked at all the other requests on the Product Board before I submitted this issue**
> Issue is known and already posted on Product Board

Hello,

I would like to use Strapi but our team need to support mobile devices.

We would love to contribute to Strapi by making it responsive.

Could we submit a PR ?

Related to
- #463 
- https://stackoverflow.com/questions/48254489/strapi-responsive-ui

**Comment by derrickmehaffy** [comment]
@felixdenoix you are absolutely welcome to submit a PR. I would recommend you talk to @soupette and @virginieky on the strapi slack though before you start working on it.

**Comment by felixdorn** [comment]
Let's go. I sent a DM to @soupette

**Comment by soupette** [comment]
That's something that could be very great for strapi but we do use some librairies that are not mobile compatible (I am thinking about `draftjs` but I am pretty sure that we could do something about it). 
I would love to see such contributions but I think that it will need some design especially for the main left menu.

**cc/** @Aurelsicoko
